src/main/java/org/zalando/fauxpas/TryWith.java

```diff
@@ -1,25 +1,31 @@
 package org.zalando.fauxpas;
 
 import lombok.SneakyThrows;
+import org.apiguardian.api.API;
 
 import javax.annotation.Nullable;
 
+import static org.apiguardian.api.API.Status.MAINTAINED;
+import static org.apiguardian.api.API.Status.STABLE;
+
 public final class TryWith {
 
     TryWith() {
         // package private so we can trick code coverage
     }
 
+    @API(status = MAINTAINED)
     public static <O extends AutoCloseable, I extends AutoCloseable, X extends Throwable> void tryWith(
             @Nullable final O outer, @Nullable final I inner, final ThrowingBiConsumer<O, I, X> consumer) throws X {
 
-        tryWith(outer, (ಠ_ಠ) -> {
-            tryWith(inner, (ツ) -> {
-                consumer.tryAccept(outer, inner);
+        tryWith(outer, a -> {
+            tryWith(inner, b -> {
+                consumer.tryAccept(a, b);
```

Comment:
does not look like equivalent change

Reply:
It is, you need to take look at the implementation of that method.